pkgs/development/python-modules/us/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, stdenv
+, jellyfish
+}:
+
+buildPythonPackage rec {
+  pname = "us";
+  version = "1.0.0";
+  name = "${pname}-${version}";
+  doCheck = !stdenv.isDarwin;
```

Comment:
why not? Add a comment when disabling tests

Reply:
I think I just copied an existing derivation - I will remove